composer.json

```diff
@@ -10,7 +10,7 @@
 		}
 	],
 	"require": {
-		"pmc/unit-test": "dev-main",
+		"pmc/unit-test": "dev-fix-mock-mailer",
```

Comment:
@mauteri Is this intentional?

Reply:
It was, but was fixed upstream so has been reverted to `dev-main`